### PR TITLE
Made changes necessary to be compatible with Elixir 1.2.x.

### PR DESCRIPTION
**Pull request by smashedtoatoms**
# What is this?

I updated dependencies and a few incompatibilities to allow Etude to work in Elixir 1.2.x.

### CODE REVIEW

**Comment by camshaft** [comment]
the .travis.yml needs to be update to use 1.2 https://travis-ci.org/camshaft/etude/jobs/100499365#L143

File: mix.exs [hunk 1]
```diff
@@ -24,14 +25,14 @@ defmodule Etude.Mixfile do
   end
 
   defp deps do
-    [{:rebind, "~> 0.1.0"},
+    [{:rebind, "~> 0.1.3"},
      {:lineo, "~> 0.1.0"},
      # hex doesn't publish github deps. i'd prefer uwiger publish parse_trans.
      # anyone that uses etude will need to add this manually for now
```
Comment (camshaft) [inline mix.exs]: can you remove this comment as well since we're now using the version from hex

Comment (smashedtoatoms) [inline mix.exs]: Oops, didn't think to verify that travis worked.  That's fixed, as is the comment cleanup.  I rebased and recommitted.

**Comment by camshaft** [comment]
beautiful

**Pull request merged**